### PR TITLE
added timesince

### PR DESCRIPTION
**Pull request by prakash87**

### CODE REVIEW

File: dmutils/filters.py [hunk 1]
```diff
@@ -45,3 +46,30 @@ def format_links(text):
         return formatted_text
     else:
         return text
+
+
+def timesince(dt, default="just now"):
+    """
+    Returns string representing "time since" e.g.
+    3 days ago, 5 hours ago etc.
+    """
```
Comment (sarneaud) [inline dmutils/filters.py:55]: We have tests for these filters in tests/test_filters.py

You should add some more for this new one.

**Comment by prakash87** [comment]
@sarneaud  tests added.

File: tests/test_filters.py [hunk 2]
```diff
@@ -95,3 +96,59 @@ def test_multiple_urls():
...
+
+        now - timedelta(days=30),
+        now - timedelta(days=60),
+
+        now - timedelta(days=365),
+        now - timedelta(days=365*2),
+    ]
+
+    texts = [
+        'just now',
```
Comment (sarneaud) [inline tests/test_filters.py:129]: This has a tiny race condition.  The test will fail if utcnow() is called just before the rollover to the next second.  Sure, this is unlikely, but if you have a few thousand test cases that are flaky, you start getting spurious build failures, which are a waste of time.

Two possible solutions:
- Make the "now" time into a parameter that you can pass into timesince()
- Use freezegun to set a fixed now time for test cases (other tests in this repo use it, so you can look around for examples)

Comment (prakash87) [reply tests/test_filters.py:129]: @sarneaud  Added 'now' parameter in timesince which might come handy in other cases as well. Thanks for the suggestion :+1:

**Pull request merged**